introduction-to-datasets.md

```diff
@@ -0,0 +1,130 @@
+
+To ensure compatibility with Hive and other underlying systems, names and namespaces in URIs must be mode of alphanumeric characters or underscore (\_) and cannot start with a number.
+
+You can create datasets in Hive, HDFS, HBase, or as local files. There are several URI patterns available and additional options. See [Dataset and View URIs][uris].
+
+[list]: {{site.baseurl}}/apidocs/org/kitesdk/data/Datasets.html#list(java.net.URI)
+[uris]:{{site.baseurl}}/URIs.html
+
+## Schemas
+
+A schema defines the field names and datatypes for a dataset. Kite relies on an Apache Avro schema definition for each dataset. Kite standardizes data definition by using Avro schemas for both Parquet and Avro, and supports the standard Avro object models _generic_ and _specific_.
+
+For example, this is the schema definition for a table using four columns from the `movies` dataset.[<sup>1</sup>](#notes)
+
+```json
+{
+  "type":"record",
+  "name":"Movie",
+  "namespace":"org.kitesdk.examples.data",
+  "fields":[
+    {"name":"id","type":"int"},
+    {"name":"title","type":"string"},
+    {"name":"release_date","type":"string"},
+    {"name":"imdb_url","type":"string"}
+  ]
+}
+```
+
+The following links provide examples for inferring schemas from data files or Java classes.
+
+| Java API                                                      | Command Line Interface |
+| --------                                                      | ---------------------- |
+| [Inferring a schema from a Java Class][api-schema-from-class] | [Inferring a schema from a Java class][cli-schema-from-class] |
+| [Using the schema of an Avro data file][api-schema-from-data] | [Inferring a schema from a CSV file][cli-schema-from-csv] |
+
+[api-schema-from-class]: {{site.baseurl}}/Inferring-a-Schema-from-a-Java-Class.html
+[api-schema-from-data]: {{site.baseurl}}/Inferring-a-Schema-from-an-Avro-Data-File.html
+[cli-schema-from-class]: {{site.baseurl}}/cli-reference.html#obj-schema
+[cli-schema-from-csv]: {{site.baseurl}}/cli-reference.html#csv-schema
+
+## Partitions
+
+Partitions define logical divisions for data storage. For example, you might most often work with data using time-based queries. You can define a partitioning strategy by year, month, and day. When you are using data from January 8, 2015, Hadoop only has to access data stored in the partition `/year=2015/month=1/day=8`. By using partitions that correspond to your most common queries, your applications run more quickly.
+
+Partitioning is optional, because there are times when partitioning is not the most efficient solution. However, you should always consider partitioning as a best practice when planning your dataset.
+
+You define your partition strategy in [partition strategy JSON format][psjf] and apply it when you create your dataset. See [Partitioned Datasets][ps].
+
+When using the API, you define your dataset in the [`DatasetDescriptor.Builder`][ddb]. 
```

Comment:
This sentence doesn't make sense on its own. What are you trying to get across here?